packages/datadog-instrumentations/src/helpers/register.js

```diff
@@ -19,16 +19,27 @@ const hooks = require('./hooks')
 const instrumentations = require('./instrumentations')
 const names = Object.keys(hooks)
 const pathSepExpr = new RegExp(`\\${path.sep}`, 'g')
+
+const convertInstrumentationName = (name) => {
+  return name.replaceAll('@', '').replaceAll('-', '').replaceAll('/', '').replaceAll('_', '').toLowerCase()
+}
+
 const disabledInstrumentations = new Set(
-  DD_TRACE_DISABLED_INSTRUMENTATIONS ? DD_TRACE_DISABLED_INSTRUMENTATIONS.split(',') : []
+  DD_TRACE_DISABLED_INSTRUMENTATIONS
+    ? DD_TRACE_DISABLED_INSTRUMENTATIONS.split(',').map(convertInstrumentationName)
+    : []
 )
+const enabledInstrumentations = new Set()
```

Comment:
I think this might be more clear:

```suggestion
const reenabledInstrumentations = new Set()
```

Specifically, it's not really a list of enabled instrumentations. There are many instrumentations that could be enabled that are outside of the list. Say, mysql or express. Instead it's a list of instrumentations that we force to be enabled in an attempt to modify their default behavior of being disabled. It's like a disdisable or undisable or reenable.